hooks/check-message.py

```diff
@@ -2,8 +2,25 @@
 
 import re, sys
 
-KEYWORDS = "(Add|Feature|Change|Remove|Codechange|Cleanup|Fix|Revert|Doc|Update|Upgrade|Prepare)"
-ISSUE = "#\d+"
+# Player facing changes:
+KEYWORDS = "(Add|"        # : Similar to Feature, but for small functionalities.
+KEYWORDS += "Change|"     # : Changing existing behavior to such an extent as to be player visible.
+KEYWORDS += "Doc|"        # : Update player-facing documentation contained in `docs/` folder or developer facing documentation in the various markdown files.
+KEYWORDS += "Feature|"    # : Adding a significant new functionality to the game. This can be small in code-size, but is meant for the bigger things from a player perspective.
+KEYWORDS += "Fix|"        # : Fixing an issue with the game (as seen by the player).
+KEYWORDS += "Remove|"     # : Completely removing a functionality.
+KEYWORDS += "Revert|"     # : Reverting an earlier Feature / Add / Change / Fix / Remove.
+KEYWORDS += "Update|"     # : Translation updates.
+
+# Developer only visible changes:
+KEYWORDS += "Codechange|" # : Changes to the code the player is not going to notice. refactors, modernization, etc.
+KEYWORDS += "Codefix|"    # : Fixing problems in earlier commits that players won't notice: wrong comments, missing files, CI changes.
+KEYWORDS += "Cleanup|"    # : Similar to Codechange, but when it is more about removing old code, rather than an actual change.
+KEYWORDS += "Prepare"     #
+KEYWORDS += "Upgrade)"    #
```

Comment:
`OpenTTD (git)  % git log --oneline --grep=Upgrade
1c738e508 Upgrade: [CI] Bump Apple-Actions/import-codesign-certs in the actions group (#12745)
62d7d92a0 Upgrade: [CI] bump the actions group with 9 updates (#11881)
173ed81db Change: [Actions] Upgrade import-codesign-certs dependency in macOS build workflow
1feba55f0 (svn r26613) -Feature: Upgrade currently active newgrfs to newest installed version.
`

Only 4 uses of this keyword ever.  2 of them not even as a prefix. Based on the usage for "Codefix"  it seems like maybe this one can be removed entirely?